scripts/settings.js

```diff
@@ -620,3 +636,28 @@ async function fillNodeSelect() {
     // And update the UI to reflect them
     doms.domNodeSelect.value = cNode.url;
 }
+
+/**
+ * Toggle Advanced Mode at runtime and in DB
+ */
+export async function toggleAdvancedMode() {
+    fAdvancedMode = !fAdvancedMode;
+
+    // Configure the app accordingly
+    await configureAdvancedMode();
+
+    // Update the setting in the DB
+    const database = await Database.getInstance();
+    await database.setSettings({ advancedMode: fAdvancedMode });
+}
+
+/**
+ * Configure the app functionality and UI for the current mode
+ */
+async function configureAdvancedMode() {
+    // Re-render the Import Input UI
+    await guiUpdateImportInput();
+
+    // Hide or Show the "Mnemonic Passphrase" in the Seed Creation modal
+    doms.domMnemonicModalPassphrase.hidden = !fAdvancedMode;
```

Comment:
Fixed this, as well as resetting it when the user changes the input key (if for some weird reason they wrote a seed phrase, then a BIP39 Passphrase, but removed the Seed Phrase for a WIF key, then it would still attempt to use the "Passphrase" value, even if hidden, while importing).